### PR TITLE
Removing defaults channel

### PR DESCRIPTION
**Pull request by gradientsky**
*Description of changes:*
Removed defaults channel since it's a bad practice

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/build_gpu.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: autogluon_py3
 channels:
 - conda-forge
-- defaults
 dependencies:
+- openmp
```
Comment (Innixma) [inline docs/build_gpu.yml:5]: why add openmp?

**Comment by sxjscience** [comment]
Will doc-failure be related to https://github.com/awslabs/autogluon/blob/70af6f34d47eb85779fdcefaf1b94c4a7d16f6da/docs/build_contrib_gpu.yml#L13 ? I previously tried d2l-book in another project and I used the following version:

```python
python -m pip install -U git+https://github.com/d2l-ai/d2l-book.git@6209db8923aca9306a02bf6b65defca1ae375ec6
```

**Comment by sxjscience** [comment]
Another reason is sphinx. I'm not sure what happened but we can also try to add sphinx in https://github.com/awslabs/autogluon/blob/master/docs/build_contrib_gpu.yml as dependencies rather than inside pip.

**Comment by szha** [comment]
Job PR-1842-12 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1842/12/index.html

**Review by sxjscience — APPROVED** [review]
Thanks!!!

**Pull request merged**